docusaurus/docs/dev-docs/api/plugins/server-api.md

```diff
@@ -130,6 +130,15 @@ module.exports = () => ({
 });
 ```
 
+To remove a cron job you need to add a cron job in the object format where the key is a unique name for the cron job.
```

Comment:
This is a bit unclear to me 🤔 (especially when you read that to remove x you need to add x, this seems counterintuitive 😅). How would you rephrase that? 

We could also add an example of a cron job defined with a unique name in the example above (e.g. "myJob" so that people know what we are talking about when we remove "myJob").